scripts/dex-router/src/const.ts

```diff
@@ -1,6 +1,6 @@
 // Time to wait in milliseconds before submitting a router bid on opportunity
 // We introduce this wait time, since this router bid is a backup to searchers normally fulfilling
-export const OPPORTUNITY_WAIT_TIME_MS = 800;
+export const OPPORTUNITY_WAIT_TIME_MS = 0;
```

Comment:
let's remove this logic completely. We can always look in git commit history and put it back if needed